controllers/rollingupgrade_controller.go

```diff
@@ -294,9 +403,10 @@ func (r *RollingUpgradeReconciler) TerminateNode(ruObj *upgrademgrv1alpha1.Rolli
 		}
 	}
 
-	log.Printf("%s: Termination output: %v\n", ruObj.Name, result)
+	log.Infof("%v: terminated instance %v", ruObj.Name, instanceID)
```

Comment:
These two lines can be consolidated into a single log message.

Reply:
I don't think we need the termination output, do we? clutters up to logs